### ISSUE TITLE
Data importing: SINFONI, FLAMES, ICUBER

### ISSUE THREAD

**Issue by brechmos-stsci** [spacetelescope/cubeviz#249]
[I talked with Jesse about this already, but so there is a ticket...]

The following three cube types all appear to have the same type of error and will probably require a coding tweak with the data importing. I asked @javerbukh to take a look at this and I went through the code and configuration files so he has a decent idea of how to do this.

SINFONI
/astro/3/jwst_da_sprint_testdata/IFU_datacubes/SINFONI/SINFONI_CAS64_k_bjames.fits
ERROR: All data appears to be in the first HDU

FLAMES
/astro/3/jwst_da_sprint_testdata/IFU_datacubes/FLAMES/FLAMES_Haro11_L6_cube_bjames.fits
/astro/3/jwst_da_sprint_testdata/IFU_datacubes/FLAMES/FLAMES_Haro11_L6_cubeerr_bjames.fits
ERROR: All data appears to be in the first HDU

ICUBER
/astro/3/jwst_da_sprint_testdata/IFU_datacubes/CWI/Cube1_Bradford/image13657_icuber.fits
ERROR: All data appears to be in the first HDU

**Comment by kassin** [comment]
Note that the ICUBER should be CWI.  CWI is the name of the instrument.  I don't know what ICUBER is.

**Comment by brechmos-stsci** [comment]
@kassin Ok re icuber/cwi. Each of those three have data in the first HDU and none are read in correctly so I think it is a similar reason.  Hence why they are all grouped together.

**Comment by javerbukh** [comment]
Data importing now works for CWI and SINFONI. Once PR #279 is merged, I will close this issue and continue working on Data Importing, specifically the WCS loading problems with FLAMES, JWST, and MUSE, in another issue.

**Comment by brechmos-stsci** [comment]
#279 is merged, closing
